modules/op-modifiable-fields-with-default-values.adoc

```diff
@@ -47,3 +47,5 @@ You can modify the default values of the following metrics fields in the `Tekton
 * `metrics.taskrun.level` (default: `task`): This field determines the level of the task run metrics. Acceptable value is `taskrun`, `task`, or `namespace`.
 
 * `metrics.pipelinerun.level` (default: `pipeline`): This field determines the level of the pipeline run metrics. Acceptable value is `pipelinerun`, `pipeline`, or `namespace`.
+
+* `coschedule (default: `workspaces`): This flag determines how `PipelineRun` pods are scheduled with the affinity assistant. Acceptable values are “workspaces” (default), “pipelineruns”, “isolate-pipelinerun”, or “disabled”.
```

Comment:
“pipelineruns”, “isolate-pipelinerun” are alpha features mentioned in the upstream doc.